### ISSUE TITLE
Things I love about your site

### ISSUE THREAD

**Issue by lucymonie** [FAC9/TheReposWebsite#49]
- Project table for the whole team creates a sense of unity
- Clean design with a unified look across site elements and lots of breathing space
- The nav menu looks like a stage curtain or an art deco feature and this hints at some personality
- Nice up and down arrow action

**Comment by tbtommyb** [comment]
+1, the rabbits text is particularly inspired!

**Comment by marisid** [comment]
I agree with the above and I love the black and white design :smile:

**Comment by SavageWilliam** [comment]
Leaving this issue 'unresolved' - makes us feel great
